batch/src/main/java/com/hartwig/batch/operations/SageGermline.java

```diff
@@ -2,132 +2,110 @@
-                localReferenceBamFile,
-                "-tumor",
-                tumorSampleName,
-                "-tumor_bam",
-                localTumorBamFile,
-                "-germline -hard_filter -hard_min_tumor_qual 0 -hard_min_tumor_raw_alt_support 3 -hard_min_tumor_raw_base_quality 30",
-                "-panel_bed",
-                panelBed,
-                "-high_confidence_bed",
-                highConfidenceBed,
-                "-hotspots",
-                hotspots,
-                "-ref_genome",
-                resourceFiles.refGenomeFile(),
-                "-out",
-                output,
-                "-threads",
-                Bash.allCpus());
+            final BashStartupScript startupScript, final RuntimeFiles executionFlags) {
+
+        final ResourceFiles resourceFiles = ResourceFilesFactory.buildResourceFiles(RefGenomeVersion.HG37);
 
-        // Download required resources
-        startupScript.addCommand(() -> format("gsutil -u hmf-crunch cp %s %s", "gs://batch-sage/resources/sage.jar", "/opt/tools/sage/pilot/sage.jar"));
-        startupScript.addCommand(() -> format("gsutil -u hmf-crunch cp %s %s", "gs://batch-sage/resources/ActionableCodingPanel.hg19.bed.gz", panelBed));
-        startupScript.addCommand(() -> format("gsutil -u hmf-crunch cp %s %s", "gs://batch-sage/resources/KnownHotspots.hg19.vcf.gz", hotspots));
-        startupScript.addCommand(() -> format("gsutil -u hmf-crunch cp %s %s", "gs://batch-sage/resources/NA12878_GIAB_highconf_IllFB-IllGATKHC-CG-Ion-Solid_ALLCHROM_v3.2.2_highconf.bed.gz", highConfidenceBed));
+        final InputFileDescriptor remoteReferenceFile = inputs.get("reference");
+        final InputFileDescriptor remoteReferenceIndex = cramIndex(remoteReferenceFile);
+        final String localReferenceFile = localFilename(remoteReferenceFile);
+        final String localReferenceBam = localReferenceFile.replace("cram", "bam");
+        final String referenceSampleName = inputs.get("referenceSample").remoteFilename();
 
-        // Download bams (in parallel)
-        startupScript.addCommand(() -> "touch /data/input/files.list");
-        startupScript.addCommand(() -> format("echo %s | tee -a  /data/input/files.list", gcTumorBamFile));
-        startupScript.addCommand(() -> format("echo %s.bai | tee -a /data/input/files.list", gcTumorBamFile));
-        startupScript.addCommand(() -> format("echo %s | tee -a /data/input/files.list", gcReferenceBamFile));
-        startupScript.addCommand(() -> format("echo %s.bai | tee -a /data/input/files.list", gcReferenceBamFile));
-        startupScript.addCommand(() -> format("cat /data/input/files.list | gsutil -m -u hmf-crunch cp -I %s", "/data/input/"));
+        // Download latest jar file
+        startupScript.addCommand(() -> format("gsutil -u hmf-crunch cp %s %s",
+                "gs://batch-sage-validation/resources/sage.jar",
+                "/opt/tools/sage/" + Versions.SAGE + "/sage.jar"));
```

Comment:
If the JAR version is stable at this point it's preferable to make an image containing it. Appreciate that during development this would be more expedient but if this code is going to stick around I'd rather it's more "official".